generator/internal/genclient/translator/openapi/openapi.go

```diff
@@ -91,33 +95,12 @@ func (t *Translator) makeAPI() (*genclient.API, error) {
 			Documentation: msg.Schema().Description,
 			Fields:        fields,
 		}
-
 		api.Messages = append(api.Messages, &message)
 	}
 	return api, nil
 }
 
-// Translates OpenAPI specification into a [genclient.GenerateRequest].
-func (t *Translator) Translate() (*genclient.GenerateRequest, error) {
-	api, err := t.makeAPI()
-	if err != nil {
-		return nil, err
-	}
-
-	codec, err := language.NewCodec(t.language)
-	if err != nil {
-		return nil, err
-	}
-	api.State = t.state
-	return &genclient.GenerateRequest{
-		API:         api,
-		Codec:       codec,
-		OutDir:      t.outDir,
-		TemplateDir: t.templateDir,
-	}, nil
-}
-
-func (t *Translator) makeMessageFields(messageName string, message *base.Schema) ([]*genclient.Field, error) {
+func makeMessageFields(messageName string, message *base.Schema) ([]*genclient.Field, error) {
```

Comment:
Soon we will need the `t *Translator` object again:

https://github.com/googleapis/google-cloud-rust/pull/71/files#diff-51632d73d29bc895749a0c2b04811475cddc5dd97a1f13b971f6509daf2dab85R304

We need to dynamically insert some messages into `t.state`.

Reply:
Will comment on that PR

Reply:
You need to rebase, there is a conflict after I merged #71.